.gitignore

```diff
@@ -10,3 +10,4 @@ build
 dist
 venv
 throughput
+.vscode/
```

Comment:
newline

Reply:
this should be in a global gitignore, not here

Reply:
true

Reply:
Ack